### ISSUE TITLE
Remove/seperate localStorage persistance

### ISSUE THREAD

**Issue by roberttod** [QubitProducts/driftwood#33]
I liked the idea of persisting to localStorage, it's particularly good for logging in production. The issue I have with it though is that if I want to programmatically pass a log flag to a module because, let's say, I am running it in debug mode, then I end up persisting that flag.

A couple of use cases a) passing a `logLevel` flag from ssg-proxy so that errors are always logged b) turning on `info` logging for when smartserve is in preview mode.

The way I see it, driftwood should not persist to localStorage by default however there should be another method for doing this and then the localStorage setting should override everything else.

Example
_smartserve.js_

``` js
// All info is logged when in preview mode
if (previewMode()) {
  driftwood.enable({ '*': 'info' })
}
```

_chrome console_

``` js
// Override the settings in preview mode so there is no logging
window.__qubit.logger.disable()
```

**Comment by roberttod** [comment]
I also think that this is more standard and aligns the node and browser logic.

**Comment by oliverwoodings** [comment]
What about:

```js
driftwood.enable({ '*': 'info' }, { persist: true })
```

or something along those lines?

**Comment by roberttod** [comment]
Yep, that would be fine. As long as this isn't the default behaviour.

**Comment by oliverwoodings** [comment]
Fixed by #27
